repose-aggregator/components/services/http-client-service/http-client-service-impl/src/main/java/org/openrepose/core/services/httpclient/impl/ProxyUtilities.java

```diff
@@ -51,15 +52,17 @@ public static SSLContext getTrustingSslContext() {
     public static class AllTrustingManager implements X509TrustManager {
         @Override
         public X509Certificate[] getAcceptedIssuers() {
-            return null;
+            return new X509Certificate[0];
```

Comment:
I have a slight concern on this and most of the other don't return null changes. Were contracts for the implemented method consulted, because null may have a very specific meaning for some of them that an empty list doesn't meet.